README.md

```diff
@@ -41,7 +42,7 @@ The command line tool is another way of converting data, as well as managing tem
 | Option | Name | Optionality | Default | Description |
 | ----- | ----- | ----- |----- |----- |
 | -d | TemplateDirectory | Required | | Root directory of templates. |
-| -r | RootTemplate | Required | | Name of root template.<br><br> Valid values for **HL7v2 to FHIR** conversion: "ADT_A01", "OML_O21", "ORU_R01", "VXU_V04". <br><br> Valid values for **C-CDA to FHIR** conversion: "CCD", "ConsultationNote", "DischargeSummary", "HistoryandPhysical", "OperativeNote", "ProcedureNote", "ProgressNote", "ReferralNote", "TransferSummary". |
+| -r | RootTemplate | Required | | Name of root template.<br><br> **HL7v2 to FHIR**: "ADT_A01", "OML_O21", "ORU_R01", "VXU_V04" <br><br> **C-CDA to FHIR**: "CCD", "ConsultationNote", "DischargeSummary", "HistoryandPhysical", "OperativeNote", "ProcedureNote", "ProgressNote", "ReferralNote", "TransferSummary" <br><br> **JSON to FHIR**: "Stu3ChargeItem", "ExamplePatient" <br> (*These are sample templates for use, not default templates that adhere to any pre-defined JSON message types. JSON does not have any standardized message types, unlike HL7v2 messages or C-CDA documents. Therefore, instead of default templates we provide you with some sample templates that you can use as a starting guide for you to modify and customize.*)  |
```

Comment:
As @ranvijaykumar pointed out, we may need to update the available templates for HL7 v2 since new templates are added. The available templates in main branch could be found here: https://github.com/microsoft/FHIR-Converter/tree/main/data/Templates/Hl7v2.